packages/examples/fortune-v2/reputation-oracle/server/src/modules/webhook/webhook.service.ts

```diff
@@ -198,13 +203,13 @@ export class WebhookService {
    * Get the oracle intermediate results at the escrow address.
    * @param chainId - Chain id.
    * @param escrowAddress - Escrow address for which results will be obtained.
-   * @returns {Promise<FinalResult[]>} - Return an array of intermediate results.
+   * @returns {Promise<any>} - Return an array of intermediate results.
    * @throws {Error} - An error object if an error occurred.
    */
   public async getIntermediateResults(
     chainId: ChainId,
     escrowAddress: string,
-  ): Promise<FinalResult[]> {
+  ): Promise<any> {
```

Comment:
Using typed objects we can improve the tests of this function and make it more reliable
```suggestion
  public async getIntermediateResults<T>(
    chainId: ChainId,
    escrowAddress: string,
  ): Promise<T> {
```